src/epsimage.cpp

```diff
@@ -317,7 +317,7 @@ void readWriteEpsMetadata(BasicIo& io, std::string& xmpPacket, NativePreviewList
 #ifdef DEBUG
   EXV_DEBUG << "readWriteEpsMetadata: First line: " << firstLine << "\n";
 #endif
-  bool matched = std::find(epsFirstLine.begin(), epsFirstLine.end(), firstLine) != epsFirstLine.end();
+  auto matched = Exiv2::find(epsFirstLine, firstLine);
```

Comment:
[discussion] `auto` here is taking the value of `T *`. Even though it is not needed, I normally find more readable to use `auto *` to declare the intention of using pointers. What do you think about this?

Reply:
I don't like auto* mostly because of const. Eg.

auto = const char*
auto* = char*
const auto = const char* const
const auto* = const char*

Regular auto makes more sense IMO.